packages/app/src/app/pages/Dashboard/Content/routes/Recent/index.tsx

```diff
@@ -1,77 +1,111 @@
-import React from 'react';
```

Comment:
this is actually the old `Home/index` page carried over, but renamed in place of the removed `Recent/index`, sorry about the extra diffs